src/pages/Refed/Refed.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Button, Container, Card } from "react-bootstrap";
+import { Link } from 'react-router-dom';
+import { toast } from "react-toastify";
+import GoogleSignIn from "../../components/GoogleSignIn/GoogleSignIn";
+import Search from "../../components/Search/Search";
+import './Refed.css';
+
+function Refed() {
+
+    function onAfterLogin() {
+        
+    }
+
+    return (
+        <div className="refed">
+            <Container>
+                <Search />
+                <Card>
+                    <Card.Header>
+                        <Card.Title>Invitation</Card.Title>
+                    </Card.Header>
+                    <Card.Body>
+                        <p>You were invited to use this application because someone thought it would be interesting and helpful to you.</p>
+                        <p>We hope you will enjoy our service.</p>
+                        <p>As a welcome gift you can received 1 day of premium to test out our premium features. You can check out all the features and your remaining time <Link to="/premium">here</Link>.</p>
+                        <hr />
+                        <p>To get access to the free premium day you have to login with google:</p>
+                        <GoogleSignIn onAfterLogin={onAfterLogin} />
+                        <Link to="/"><Button>Go to main page</Button></Link>
```

Comment:
```suggestion
                        <p>We use google accounts because they are more secure than requiring a separate login. We use your google Id and email in compliance with our <a href="https://coflnet.com/privacy">privacy policy</a> (i.e. to know what settings you made and contact you in case we need to)</p>
                        
                        <Link to="/"><Button>Go to main page</Button></Link>
```